CHANGES.md

```diff
@@ -1,5 +1,488 @@
+* Added a new function `removeAllCoordinateSystems()` in `PCLVisualizer`
+  [[#965]](https://github.com/PointCloudLibrary/pcl/pull/965)
+* Made `PCLVisualizer::addPointCloudPrincipalCurvatures()` templated on point
+  and normal type
+  [[#965]](https://github.com/PointCloudLibrary/pcl/pull/965)
+* Fixed a minor bug in `PCLVisualizer::updatePolygonMesh()`
+  [[#977]](https://github.com/PointCloudLibrary/pcl/pull/977)
+* Fixed a minor bug in `ImageViewer::addMask()`
+  [[#990]](https://github.com/PointCloudLibrary/pcl/pull/990)
+* Fixed opacity handling in `ImageViewer`
+  [[#995]](https://github.com/PointCloudLibrary/pcl/pull/995)
+* Fixed a bug with `ImageViewer` not displaying anything with VTK 6
+  [[#1009]](https://github.com/PointCloudLibrary/pcl/pull/1009)
+* Updated `ImageViewer` to work around a bug in VTK 6.1
+  [[#1017]](https://github.com/PointCloudLibrary/pcl/pull/1017)
+* Fixed an Eigen-related compilation error in `PCLVisualizer::renderView()`
+  [[#1019]](https://github.com/PointCloudLibrary/pcl/pull/1019)
+* Fixed wrong axis flipping in `PCLVisualizer::renderView()`
+  [[#1026]](https://github.com/PointCloudLibrary/pcl/pull/1026)
+* Fixed a bug in `renderViewTesselatedSphere` when generated vertices were not
+  guaranteed to stay on the unit sphere
+  [[#1043]](https://github.com/PointCloudLibrary/pcl/pull/1043)
+* Fixed misaligned context items in `ImageViewer`
+  [[#1049]](https://github.com/PointCloudLibrary/pcl/pull/1049)
+* Fixed opacity handling for layered rectangles of context items in
+  `ImageViewer`
+  [[#1051]](https://github.com/PointCloudLibrary/pcl/pull/1051)
+* Fixed a regression in `RenderViewsTesselatedSphere::generateViews()` related
+  to handling of multiple VTK versions
+  [[#1056]](https://github.com/PointCloudLibrary/pcl/pull/1056)
+  [[#1067]](https://github.com/PointCloudLibrary/pcl/pull/1067)
+  [[#1072]](https://github.com/PointCloudLibrary/pcl/pull/1072)
+* Updated `PCLVisualizer` to use `PointCloudColorHandlerRGBAField` for
+  `PointXYZRGBA` clouds by default
+  [[#1064]](https://github.com/PointCloudLibrary/pcl/pull/1064)
+* Fixed a bug in `PointCloudColorHandlerLabelField` where red and blue channels
+  were swapped
+  [[#1076]](https://github.com/PointCloudLibrary/pcl/pull/1076)
+* Updated `PCLPlotter` to ignore NaN values in histogram computation
+  [[#1120]](https://github.com/PointCloudLibrary/pcl/pull/1120)
+  [[#1126]](https://github.com/PointCloudLibrary/pcl/pull/1126)
+* Fixed initial size of the `PCLVisualizer` window
+  [[#1125]](https://github.com/PointCloudLibrary/pcl/pull/1125)
+* Changed default representation of all shapes in `PCLVisualizer` to "surface"
+  [[#1132]](https://github.com/PointCloudLibrary/pcl/pull/1132)
+* Added a check for model coefficients size in functions that add shapes to
+  `PCLVisualizer`
+  [[#1142]](https://github.com/PointCloudLibrary/pcl/pull/1142)
+* Added a new variant of `PCLVisualizer::addModelFromPolyData()` that allows to
+  display colored meshes
```

Comment:
You should remove this line because I have removed the function in https://github.com/PointCloudLibrary/pcl/pull/1241; it was not needed. #1211 summarizes the changes.